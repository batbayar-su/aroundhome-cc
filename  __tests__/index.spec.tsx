import { render } from '@testing-library/react'
import IndexPage from '../pages/index'

it('renders IndexPage', () => {
  const { container } = render(<IndexPage />)
  expect(container).toMatchSnapshot()
})