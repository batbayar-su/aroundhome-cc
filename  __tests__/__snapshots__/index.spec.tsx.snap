// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders IndexPage 1`] = `
<div>
  <div>
    Something went wrong
  </div>
</div>
`;
